postgres/src/test/java/org/jdbi/v3/pg/TestSqlArrays.java

```diff
@@ -87,7 +98,7 @@ public void testIntList() throws Exception {
         UUID[] fetchUuidArray();
 
         @SqlUpdate(U_INSERT)
-        void insertUuidArray(UUID[] u);
+        void insertUuidArray(@Bind("uuids") UUID[] u);
```

Comment:
Do I need to compile the class with the debug information to make that work? Currently I get
```
java.lang.UnsupportedOperationException: A parameter was not given a name, and parameter name data is not present in the class file, for: public abstract void org.jdbi.v3.pg.TestSqlArrays$ArrayObject.insertUuidArray(java.util.UUID[]) :: java.util.UUID[] arg0
```